packages/server/package.json

```diff
@@ -21,8 +21,8 @@
         "dev:debug": "nodemon --exec \"node -r ts-node/register --inspect-brk\" src/index.ts | pino-pretty",
         "lint": "eslint src --ext js,ts",
         "lint:fix": "eslint src --ext js,ts --fix",
-        "test": "mocha -r ts-node/register test/**/*.ts --exit",
-        "test:debug": "mocha -r ts-node/register --inspect-brk test/**/*.ts --exit"
+        "//": "Must --forceExit due to supertest not closing https://github.com/visionmedia/supertest/issues/437",
```

Comment:
Lol I've never seen this method of commenting in JSON

Reply:
me neither 😂 

https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install